src/components/Form.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import Button from './myButton';
+import { addBook } from '../redux/books/booksSlice';
+
+function Form() {
+  const dispatch = useDispatch();
+  const [title, setTitle] = useState('');
+  const [author, setAuthor] = useState('');
+  const handelSubmit = (e) => {
+    const data = {
+      title,
+      author,
+      id: uuidv4(),
+    };
+    e.preventDefault();
+    dispatch(addBook(data));
```

Comment:
[OPTIONAL] It would be great idea, if the input field became epty after I click submit button, and this feature give users more help :)